buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -242,16 +245,42 @@ class ClusterFormationTasks {
                 esEnv['JAVA_OPTS'] += ' -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=8000'
             }
 
+            // Due to how ant exec works with the spawn option, we lose all stdout/stderr from the
+            // process executed. To work around this, when spawning, we wrap the elasticsearch start
+            // command inside another shell script, which simply internally redirects the output
+            // of the real elasticsearch script. This allows ant to keep the streams open with the
+            // dummy process, but us to have the output available if there is an error in the
+            // elasticsearch start script
```

Comment:
We used to dump the log file ES creates (see below). You are correct that this is a little better. Not only do we get jvm output that does not go there, but we also get any errors in the ES shell script itself. Essentially we are running a mini shell so that we can run ES as if it was run in the foreground. This is why the logic to read and print the log file on error below is changed to only print the file output by this shell script.